test/acceptance/kafo_configure_test.rb

```diff
@@ -5,6 +5,7 @@ module Kafo
     before do
       generate_installer
       add_manifest
+      @config_file = "#{INSTALLER_HOME}/config/installer-scenarios.d/default.yaml"
```

Comment:
I think there's already a constant `KAFO_CONFIG` for this:
https://github.com/theforeman/kafo/blob/dd1f1f73548f1e2a37e1ffc9fdf3f6d977f2d0bd/test/acceptance/test_helper.rb#L9